tidb-cloud/migrate-from-mysql-using-aws-dms.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Migrate MySQL-Compatible Databases to TiDB Cloud Using AWS DMS
+summary: Learn how to migrate data from MySQL-compatible databases into TiDB Cloud using AWS Database Migration Service (AWS DMS).
+---
+
+# Migrate MySQL-Compatible Databases to TiDB Cloud Using AWS DMS
+
+AWS Database Migration Service (AWS DMS) is a cloud service that makes it easy to migrate relational databases, data warehouses, NoSQL databases, and other types of data stores. You can use AWS DMS to migrate your data into TiDB Cloud.
+
+This document uses Amazon Relational Database Service (RDS) as an example to show how to migrate data to TiDB Cloud using AWS DMS. The procedure also applies to migrating data from self-built MySQL databases and Amazon Aurora.
+
+In this example, the data source is Amazon RDS, and the data destination is a Dedicated Tier cluster in TiDB Cloud. Both upstream and downstream databases are in the same region.
+
+## Prerequisites
+
+Before you start the migration, make sure you have read the following:
+
+- If the source database is Amazon RDS or Aurora, you need to set the `binlog_format` parameter to `ROW`. If the database uses the default parameter group, the `binlog_format` parameter is set to `MIXED` by default and cannot be modified. In this case, you need to [create a new parameter group](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_GettingStarted.Prerequisites.html#CHAP_GettingStarted.Prerequisites.params), for example `newset`, and set `binlog_format` to `ROW`. Then, modify the default parameter group to `newset`. Note that modifying the parameter group will restart the database.
+- The default collation for the utf8mb4 character set in TiDB is `utf8mb4_bin`, in MySQL 5.7 it is `utf8mb4_general_ci`, and in MySQL 8.0, it is `utf8mb4_0900_ai_ci`. If the default collation is used in the upstream MySQL, because TiDB is not compatible with it, AWS DMS cannot create the target table in TiDB and cannot migrate the data. If you encounter such a problem, you can modify the collation of the source database to `utf8mb4_bin`.
```

Comment:
delete ”in MySQL 5.7 it is `utf8mb4_general_ci` “ because
 TiDB has supported it